src/plugins/kibana_react/public/table_list_view/table_list_view.tsx

```diff
@@ -38,26 +37,61 @@ import {
   EuiConfirmModal,
   EuiCallOut,
 } from '@elastic/eui';
-
 import { npStart } from 'ui/new_platform';
+import { SavedObject } from 'ui/saved_objects/saved_object';
```

Comment:
These two `ui/*` imports are not allowed here. You can pass those to the component through props or context instread.